### DISCUSSION TITLE
Add new "Option" in CLI command to download TOC Files

### DISCUSSION THREAD

**Discussion by jkHCBB** [CMSgov/price-transparency-guide-validator#105] [Ideas]
When running the TOC schema we are asked mid process if we wish to download the files within the TOC Json file. 
I would like to ask for a new feature that allows us to use an option to specify yes or no in the initial command, so that it can run seamlessly in a pipeline. I can work around this for my automation for now but I would really like to see an integrated way instead.

Example:
cms-mrf-validator validate 2023-09-07_Diabetes---Endocrinology-Center-of-Ohio--Inc-_index.json v1.0.0 -o table_of_contents_results.txt -t table-of-contents

info  Running validator container...
info  Table of contents refers to these files:
info  == In-Network Rates ==
info  * https://mrftest.healthcarebluebook.com/DIABETES&ENDOCRINOLOGYCENTEROFOHIO/272423
info  == Allowed Amounts ==
info  * https://mrftest.healthcarebluebook.com/DIABETES&ENDOCRINOLOGYCENTEROFOHIO/281134 

_**Would you like to validate these files? [y/n]: y**_
_**Data file size is unknown. Download this file? [y/n]: y**_

info  File: https://mrftest.healthcarebluebook.com/DIABETES&ENDOCRINOLOGYCENTEROFOHIO/272423 
info  Beginning download... 
info  Downloaded 100% of file (4860 bytes) 
info  Download complete. 
info  Running validator container... 
info  File: https://mrftest.healthcarebluebook.com/DIABETES&ENDOCRINOLOGYCENTEROFOHIO/281134 
info  Beginning download... 
info  Downloaded 100% of file (142843 bytes) 
info  Download complete. 
info  Running validator container...

I would like to see an option like this:
cms-mrf-validator validate 2023-09-07_Diabetes---Endocrinology-Center-of-Ohio--Inc-_index.json v1.0.0 -o table_of_contents_results.txt -t table-of-contents **_-v/-validate yes_**

info  Running validator container...
info  Table of contents refers to these files:
info  == In-Network Rates ==
info  * https://mrftest.healthcarebluebook.com/DIABETES&ENDOCRINOLOGYCENTEROFOHIO/272423
info  == Allowed Amounts ==
info  * https://mrftest.healthcarebluebook.com/DIABETES&ENDOCRINOLOGYCENTEROFOHIO/281134 

info  File: https://mrftest.healthcarebluebook.com/DIABETES&ENDOCRINOLOGYCENTEROFOHIO/272423 
info  Beginning download... 
info  Downloaded 100% of file (4860 bytes) 
info  Download complete. 
info  Running validator container... 

info  File: https://mrftest.healthcarebluebook.com/DIABETES&ENDOCRINOLOGYCENTEROFOHIO/281134 
info  Beginning download... 
info  Downloaded 100% of file (142843 bytes) 
info  Download complete. 
info  Running validator container...

**Comment by mint-thompson** [comment]
Thank you for this suggestion! I had already started work on a feature like this a little while ago, since I guessed that there would be situations like this. Since you've asked about it, I'll try to prioritize completing it.

**Reply by jkHCBB** [reply]
Thank you!

**Reply by jkHCBB** [reply]
Hello again, any updates on this request? Thanks again!

**Reply by mint-thompson** [reply]
Sorry for not providing any updates for awhile! This feature is now available in version 2.0.0, which has just been released.

**Reply by jkHCBB** [reply]
Thank you!
